tensorflow/training/docker/2.3.0/py3/cu102/Dockerfile.gpu

```diff
@@ -27,13 +27,13 @@ ARG PIP=pip3
 ARG PYTHON_VERSION=3.7.7
 ARG OPENSSL_VERSION=1.1.1g
 
-ARG TF_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/tensorflow/r2.3_aws/20200806-080706/gpu/py37/tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl
+ARG TF_URL=s3://aws-tensorflow-binaries/tensorflow/r2.3_aws/20200902-124016/gpu/py37/tensorflow_gpu-2.3.0-cp37-cp37m-manylinux2010_x86_64.whl
```

Comment:
This change wasn't tested since tf 2.2 was build during tests execution.
[tf/buildspec.yml ](https://github.com/aws/deep-learning-containers/blob/d4cd2edf54c6d926e4a8c9a873e5c694f52b3f03/tensorflow/buildspec.yml#L4)